paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -317,13 +323,47 @@ public class PaparazziSdk @JvmOverloads constructor(
         (view.parent as ViewGroup).removeView(view)
       }
       AnimationHandler.sAnimatorHandler.set(null)
-      recomposerWatcher.unregister()
       if (hasComposeRuntime) {
         forceReleaseComposeReferenceLeaks()
       }
     }
   }
 
+  private fun renderForIdleRecomposer(
+    recomposer: Any?,
+    frame: Int,
+    additionalRenderCount: Int,
```

Comment:
Are we planning on adding a future api to allow users to configure this and the frame parameter?

Reply:
That was the initial idea but I can strip this down for now